graphql-cli/src/main/java/io/ballerina/graphql/cmd/Utils.java

```diff
@@ -65,7 +66,7 @@ public class Utils {
      * @return               the `GraphQLSchema` instance
      */
     public static Constructor getProcessedConstructor() {
-        Constructor constructor = new Constructor(Config.class);
+        Constructor constructor = new Constructor(Config.class, new LoaderOptions());
```

Comment:
Why we need this change?

Reply:
The `SnakeYaml` version `1.33` has vulnerabilities and with Java-17 the version `2.0` has been suggested to use. The version `2.0` provides the constructor which requires a parameter for load operations configuration.